Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -186,6 +259,54 @@
         "CannotFindBuilds": "Cannot find builds",
         "JenkinsNoCommitsToFetch": "Start and end build Ids are same. Commits and workitems will not be downloaded.",
         "InvalidJenkinsBuildNumber": "Cannot parse the Jenkins Build number.",
-        "InvalidJenkinsStartBuildNumber": "Invalid value mentioned for Jenkins start build number."
+        "InvalidJenkinsStartBuildNumber": "Invalid value mentioned for Jenkins start build number.",
+        "ClientIdCannotBeEmpty": "clientId must be a non empty string.",
+        "DomainCannotBeEmpty": "domain must be a non empty string.",
+        "SecretCannotBeEmpty": "secret must be a non empty string.",
+        "armUrlCannotBeEmpty": "arm Url must be a non empty string.",
+        "authorityUrlCannotBeEmpty": "authority must be a non empty string.",
+        "CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: %s, status message: %s",
+        "CallbackCannotBeNull": "callback cannot be null.",
```

Comment:
"Callback cannot be null" is not a user friendly message. The message should help the user with what inputs they need to specify or what action they can take. Please review and update messages that have type string, null etc. If these are debug messages, they don't need to be localized at all, so you can remove them from the task.json